ChakraCore Samples/Hello World/Python/README.md

```diff
@@ -0,0 +1,24 @@
+# ChakraCore Hello-world Python Sample
+This sample shows how to embed ChakraCore with [JavaScript Runtime (JSRT) APIs](http://aka.ms/corejsrtref), and complete the following tasks:
+
+1. Initializing the JavaScript runtime and creating an execution context.
+2. Running scripts and handling values.
+3. Printing out script results.
+
+## Requirements
+This sample requires Python 2.7+ is installed.
+
+## Build and run the sample
+1. Follow the instructions to build ChakraCore (https://github.com/Microsoft/ChakraCore/wiki/Building-ChakraCore)
+2. Update the sample `chakraCore = CDLL("./BuildLinux/Debug/libChakraCore.so")` with the location of `CharakCore.dll` or .so file
+3. Run the sample by `python helloWorld.py`
+
+## Remarks
+While working with Python's `CDLL/ctypes` keep in mind the [ChakraCore memory management](https://github.com/Microsoft/ChakraCore/wiki/JavaScript-Runtime-%28JSRT%29-Overview#memory-management).
+You may not hold a reference to a JsValue unless that is explicitly being referenced or has a counter on it.
```

Comment:
You may not hold reference to a JavaScript value unless it is explicitly referenced on the runtime's heap or stack, or preserved through [JsAddRef](https://github.com/Microsoft/ChakraCore/wiki/JsAddRef).